dspace-api/src/main/java/org/dspace/core/Email.java

```diff
@@ -607,7 +664,7 @@ public static void main(String[] args) {
             System.out.println(" - To: " + to);
             System.out.println(" - Subject: " + subject);
             System.out.println(" - Server: " + server);
-            boolean disabled = config.getBooleanProperty("mail.server.disabled", false);
+            boolean disabled = isMailServerDisabled(config);
```

Comment:
`mail.server.disabled` should prevail. The expected behavior should be, If true no emails are to be sent. It's what it's expected to do.